test-fixtures/update.py

```diff
@@ -1,17 +1,13 @@
 import os
-import re
-import sys
-import pathlib
 from shutil import copyfile
 from ruamel.yaml import YAML
 
 
 import requests
-import semver
 from requests.exceptions import HTTPError
 
 
-BITRISE_STACK_INFO = 'https://app.bitrise.io/app/6c06d3a40422d10f/all_stack_info'
+BITRISE_STACK_INFO = 'https://api.github.com/repos/bitrise-io/stacks/contents/content/stack_reports'
```

Comment:
I can fetch the info of this directory via Github's API.

https://github.com/bitrise-io/stacks/tree/main/content/stack_reports